crates/wit-bindgen-wasmcloud-provider-binary/.gitignore

```diff
@@ -0,0 +1,5 @@
+/target/
+target/
+.idea
+.direnv
+.envrc
```

Comment:
Is there a reason to add this on a sub-directory level and not reuse the root one?